prov/rxd/src/rxd_ep.c

```diff
@@ -453,7 +455,7 @@ struct rxd_x_entry *rxd_tx_entry_init(struct rxd_ep *ep, const struct iovec *iov
 	memcpy(&tx_entry->iov[0], iov,
 	       sizeof(*iov) * iov_count);
 
-	memset(&tx_entry->cq_entry, 0, sizeof(tx_entry->cq_entry));
+	tx_entry->cq_entry.data = 0;
```

Comment:
Trivial, but I think we can save an instruction with
`tx_entry->cq_entry.data = (flags & FI_REMOTE_CQ_DATA) ? data : 0`

Reply:
Is this setting up the CQ entry for a transmit request?

Reply:
Yes, I'll post an update

Reply:
Yes, that sounds like a good idea

Reply:
The cq data for a transmit shouldn't matter.